### ISSUE TITLE
problems formatting fat arrow functions

### ISSUE THREAD

**Issue by IgorMinar** [angular/clang-format#7]
clang-format@1.0.12 insists that this is the right way to format the following code:

```
module.exports = function makeNodeTree(destinationPath) {

  var traceurCompatibleTsModulesTree = replace(modulesTree, {
    replaceWithPath: (path, content) => {
      return content;
}
});
}
;
```

removing the fat arrow section or the return statement from the fat arrow block makes things work fine again.

**Comment by vicb** [comment]
An other occurence in https://github.com/angular/angular/blob/863eb3c559e8c175885ceb0e2d5b4e5c535bf467/modules/angular2/src/render/dom/events/key_events.ts#L91

**Comment by mprobst** [comment]
See http://reviews.llvm.org/D9906.

**Comment by mprobst** [comment]
Fixed with the new release in 5242c1ef85.
